WordPress/Classes/ViewRelated/Blog/BlogDetailHeaderView.m

```diff
@@ -63,6 +66,12 @@ - (void)setBlog:(Blog *)blog
         UIDropInteraction *dropInteraction = [[UIDropInteraction alloc] initWithDelegate:self];
         [self.blavatarDropTarget addInteraction:dropInteraction];
     }
+    
+    NSString *localizedLabel =
+        NSLocalizedString(@"Site named %@ located at %@",
```

Comment:
This label sounds a bit too difficult to understand.
What about we simplify it to something like: `@"%@, at %@"`?
Also with the `,` so the screen reader take a breath :)

Reply:
Great suggestion! Changed in 2b02be30b52bc85739a5f0d6819ac500c4f6088a.

Also, VoiceOver does not seem to read the domain very well. Do you know if there's any way to improve that?